default-views/account/register.hbs

```diff
@@ -21,5 +21,57 @@
     {{> account/register-form}}
   {{/if}}
 </div>
+<script src="https://raw.githubusercontent.com/nowsecure/owasp-password-strength-test/master/owasp-password-strength-test.js"
+        defer></script>
+<script>
+    function validatePasswordBeforeSubmit (e) {
+        e.preventDefault();
+        const pwdErrorDiv = document.getElementById('passwordHelp');
+        let pw = document.getElementById('password').value;
+        let owaspCheck = owaspPasswordStrengthTest.test(pw)
+        if (owaspCheck.strong === true) {
```

Comment:
Is this necessary? `owaspCheck.strong` will always be truish in the case that it is valid, no?

Reply:
It won't be always true. If the owasp tests failed it will return false